### ISSUE TITLE
App Crashing after using the library

### ISSUE THREAD

**Issue by AbhishekChourasia16690** [calvinchankf/CCKFNavDrawer#3]
Hello,
Thank you for the small size code base for such a big feature.
I need to support it for app with deployment target above 8.
Also my app supports only portrait mode.

I get this crash once I integrate it
-[UINavigationController setCCKFNavDrawerDelegate:]: unrecognized selector sent to instance

Could you please let me know the reason.

Thanks.

**Comment by calvinchankf** [comment]
Hi,

I am sorry to tell you that this project has been discontinued.

Calvin

**Comment by AbhishekChourasia16690** [comment]
Yes I know but the library works fine for now. But I found one strange issue with the library for iOS 8 only.
If there is any button below the hamberg menu button (from which sliding happens) which is touching the bottom of screen that button method gets called.

It would be really helpful, if you could tell me the reason for this.

Waiting for your reply.
Thanks
